src/main/java/games/strategy/engine/lobby/server/ModeratorController.java

```diff
@@ -60,39 +62,27 @@ public boolean isPlayerAdmin(final INode node) {
     return user != null && user.isAdmin();
   }
 
-  @Override
-  public void banMac(final INode node, final Date banExpires) {
-    banMac(node, banExpires != null ? banExpires.toInstant() : null);
+  private Moderator getModeratorForNode(final INode node) {
+    return new Moderator(node.getName(), node.getAddress(), getNodeMacAddress(node));
```

Comment:
If a Moderator object can be constructed entirely from an INode, you might want to consider overloading the constructor to accept an INode.

Reply:
A `Moderator` is not only constructed from information obtained from an `INode`.  The tests also construct instances, but use generated values rather than an `INode`.

In addition, an `INode` doesn't provide the MAC; that needs to be obtained from an `IServerMessenger`.

Reply:
Thanks for the clarification.
Looked like it, because all 3 parameters were directly related to the inode instance.